libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFAuthenticationManager.m

```diff
@@ -76,6 +77,10 @@
 // Oauth
 static NSString * const kSFUserAccountOAuthLoginHostDefault = @"login.salesforce.com"; // last resort default OAuth host
 static NSString * const kSFUserAccountOAuthLoginHost = @"SFDCOAuthLoginHost";
+static NSString * const kSFAuthAccessToken = @"access_token";
```

Comment:
Might I suggest making these `extern` keys instead?  I could see situations where these constants can be useful to a consuming app.